docs/src/index.md

```diff
@@ -1,23 +1,10 @@
-# Extremes for Julia documentation
 
-## High level API
+# Extreme value analysis in Julia.
 
-```@docs
-getcluster(y::Array{<:Real,1}, u₁::Real , u₂::Real=0.0)
-getcluster(df::DataFrame, u₁::Real, u₂::Real=0.0)
-gevfitbayes(y::Array{<:Real}; warmup::Int=0, niter::Int=1000, thin::Int=1, stepSize::Array{<:Real,1}=[.1,.1,.05])
-gevfit(y::Array{T,1} where T<:Real)
-gevfit(y::Array{Float64,1}, location_covariate::Array{Float64,1}; initialvalues::Array{Float64,1}=Float64[])
-gpdfit(y::Array{T} where T<:Real; threshold::Real=0.0)
-gpdfitbayes(data::Array{Float64,1}; threshold::Real=0, niter::Int = 10000, warmup::Int = 5000,  thin::Int = 1, stepSize::Array{<:Real,1}=[.1,.1])
-```
+The *Extremes.jl* package provides several functions and methods for the analysis of extreme values in Julia. In particular, methods for block maxima and threshold exceedances models are implemented, such as:
```

Comment:
Je suggère "The *Extremes.jl* package provides exhaustive high-performance functions for the analysis of extreme values in Julia."